.github/workflow_scripts/test_tabular.sh

```diff
@@ -28,7 +28,9 @@ fi
 cd tabular/
 if [ -n "$ADDITIONAL_TEST_ARGS" ]
 then
-    python -m pytest --junitxml=results.xml --runslow "$ADDITIONAL_TEST_ARGS" tests
+    # TODO: Temporary re-arrangement of tests to unblock timeout on platform tests
```

Comment:
A more effective method would be to utilize hooks from conftest.py, available [here](https://github.com/autogluon/autogluon/blob/ab9f9e1762453d7a4984227addfc91758f772048/tabular/tests/conftest.py#L50), to conduct specific tests by order. While I wouldn't advise this as a permanent solution, it is sufficient to proceed with our 1.1 release. In order to follow up on this, please create an issue for tracking comprehensive fixes in the future.